src/core/mac/mac_frame.hpp

```diff
@@ -428,6 +431,15 @@ class Frame : public otRadioFrame
      */
     void SetAckRequest(bool aAckRequest);
 
+    /**
+     * This method indicates whether or not the PanId Compression bit is set.
+     *
+     * @retval TRUE   If the PanId Compression bit is set.
+     * @retval FALSE  If the PanId Compression bit is not set.
+     *
+     */
+    bool GetPanIdCompression(void) const { return (GetFrameControlField() & kFcfPanidCompression) != 0; }
```

Comment:
what's the principle to name a method which returns a boolean?
Here the name starts with `Get`, below starts with `Is` (like `IsSrcPanIdPresent`), better to be consistent?